test_wrapper.jl

```diff
@@ -0,0 +1,95 @@
+module TestWrapper
+using Pkg
+
+function parse_file_line(failed_line)
+    r = r"(\e\[91m\e\[1m)?Test Failed(\e\[22m\e\[39m)? at (\e\[39m\e\[1m)?(?<path>[^\s\e]+)(\e\[22m)?"
```

Comment:
Are those escaped things colours? If so, could you add a brief note about that in a comment above it, please?